docs/usage.md

```diff
@@ -1,30 +1,53 @@
+For an example of generating and distributing a projection interop assembly as a NuGet package, see the [projection sample](https://github.com/microsoft/CsWinRT/tree/master/src/Samples/Net5ProjectionSample) and [associated walkthrough](https://docs.microsoft.com/windows/uwp/csharp-winrt/net-projection-from-cppwinrt-component).
 
-```  
+### Generating an interop assembly
+
+Component authors create a projection project, which is a C# library project that references the C#/WinRT NuGet package and the project-specific `.winmd` files you want to project, whether through a NuGet package reference, project reference, or direct file reference.
+
+The following project fragment demonstrates a simple invocation of C#/WinRT to generate projection sources for types in the "Contoso" namespace. These sources are then included in the project build. By default, the **Windows** and **Microsoft** namespaces are not projected.
+
+```xml  
 <PropertyGroup>
   <CsWinRTIncludes>Contoso</CsWinRTIncludes>
 </PropertyGroup>
 ```
 
-To further customize C#/WinRT behavior, refer to the [C#/WinRT NuGet documentation](../nuget/readme.md).
+For a full list of C#/WinRT NuGet project properties, refer to the [NuGet documentation](../nuget/readme.md).
+
+In the example diagram below, the projection project invokes **cswinrt.exe** at build time, which processes `.winmd` files in the "Contoso" namespace to generate projection source files and compiles these into an interop projection assembly named `Contoso.projection.dll`. The interop assembly is typically distributed along with the implementation assemblies (`Contoso.*.dll`) as a NuGet package. 
+
+<img alt="Creating projection"
+    src="images/Diagram_CreateProjection.jpg"
+    width="70%" height="50%">
+
+### Distributing the interop assembly
+
+An interop assembly is typically distributed along with the implementation assemblies as a NuGet package for application projects to reference. This package will require a dependency on C#/WinRT to include `WinRT.Runtime.dll` for .NET 5+ targets. If the interop assembly is not distributed as a NuGet package, an application project adds references to both the component interop assembly produced above and to C#/WinRT to include the `WinRT.Runtime.dll` assembly. If a third party WinRT component is distributed without an official interop assembly, an application project may add a reference to C#/WinRT to generate its own private component interop assembly.  There are versioning concerns related to this scenario, so the preferred solution is for the third party to publish an interop assembly directly.
```

Comment:
Should we link to docs? E.g.

"There are [versioning concerns](https://github.com/microsoft/CsWinRT/blob/ad6adcb997b40cea649cf4156451316e714bc3ea/docs/versioning.md) related ..."

Reply:
Not the exact same versioning concerns as that doc.  Versioning concerns here refer to different versions of the same private component.